UNRELEASED.md

```diff
@@ -1,11 +1,14 @@
 # Unreleased
-
 > Place your changes below this line.
 
 **Added:**
 
 - Added lint check for compose component usage
 
+**Fixed:**
```

Comment:
This is an interesting one with versioning. should this be a major bump? or is because compose gets bumped many majors fine to define it as a patch here? :D